src/test/resources/predicates.adoc

```diff
@@ -17,7 +17,7 @@ will be transpiled to
 
 [source,cypher,id=t1_0_expected]
 ----
-MATCH (:p)
+MATCH (p:p)
```

Comment:
Is this now mandatory? 
What happens if I have (two different) Person label nodes? like `(:Person)-[:DIRECTED]->(m)<-[:DIRECTED]-(:Person)` would it use the same `person` for both or different ones? I.e. when I actually don't care about the nodes themselves just the existence of that pattern around that movie?

Just curious.

Reply:
The Cypher-DSL does not mandate using an identifier. But this change here is essentially this:

https://github.com/neo4j-contrib/sql2cypher/pull/32/files#diff-3d380b17c4f5e148cc96786d7bd3619316d214da233d6b7b029f2c32065f573aR551

It will always name a pattern, either by the alias of the table (which was already there) or by the full name of the table.

This is to avoid https://github.com/neo4j-contrib/cypher-dsl/issues/585#issuecomment-1404751314 this kind of confusion.

The example you linked @jexp is kinda impossible to create with the translator anyhow I guess, as the table name will always go into the node.
At least until you do something different when evaluating joins, which would also completely possible.

Reply:
> What happens if I have (two different) Person label nodes? like `(:Person)-[:DIRECTED]->(m)<-[:DIRECTED]-(:Person)` would it use the same `person` for both or different ones?

Assuming this is about the SQL transpiler, you can't have two different instance of `person` table references without aliasing at least one of them explicitly. This whole discussion was about what to do with SQL table references that aren't aliased.